### PR TITLE
comments

### PR DESCRIPTION
**Pull request by chriku**

### CODE REVIEW

File: github/src/main/kotlin/gropius/sync/github/Outgoing.kt [hunk 1]
```diff
@@ -289,6 +291,28 @@ class Outgoing(
         }
     }
 
+    /**
+     * Mutate an RemovedLabelEvent upto GitHub
```
Comment (nk-coding) [inline github/src/main/kotlin/gropius/sync/github/Outgoing.kt:295]: this doc sounds wrong

File: github/src/main/kotlin/gropius/sync/github/Outgoing.kt [hunk 2]
```diff
@@ -289,6 +291,28 @@ class Outgoing(
...
+     * @param imsProjectConfig active config
+     * @param issueInfo info of the issue containing the timeline item
+     * @param userList users that have contributed to the event
```
Comment (nk-coding) [inline github/src/main/kotlin/gropius/sync/github/Outgoing.kt:298]: does not exist, replace with doc for user?

File: github/src/main/kotlin/gropius/sync/github/Outgoing.kt [hunk 3]
```diff
@@ -312,6 +337,26 @@ class Outgoing(
...
+     * @param issueInfo info of the issue containing the timeline item
+     * @param timeline TimelineItems for this issue
+     * @return List of functions that contain the actual mutation executors
+     */
+    private suspend fun pushComments(
+        imsProjectConfig: IMSProjectConfig, issueInfo: IssueInfo, timeline: List<TimelineItem>
+    ): List<suspend () -> Unit> {
+        val collectedMutations = mutableListOf<suspend () -> Unit>()
+        val relevantTimeline = timeline.mapNotNull { it as? IssueComment }
+            .filter { timelineEventInfoRepository.findByNeo4jId(it.rawId!!) == null }.forEach {
```
Comment (nk-coding) [inline github/src/main/kotlin/gropius/sync/github/Outgoing.kt:352]: wouldn't it be possible to replace the foreach with a map and not use the temp mutable list?

**Review by nk-coding — CHANGES_REQUESTED** [review]

File: github/src/main/kotlin/gropius/sync/github/Outgoing.kt (hunk 3 above)
Comment (chriku) [reply github/src/main/kotlin/gropius/sync/github/Outgoing.kt:352]: there may be more than one element added

File: github/src/main/kotlin/gropius/sync/github/Outgoing.kt (hunk 2 above)
Comment (chriku) [reply github/src/main/kotlin/gropius/sync/github/Outgoing.kt]: 3e4f88b87b2b5fefa5110c93c35b9bea1b3bbd39

File: github/src/main/kotlin/gropius/sync/github/Outgoing.kt (hunk 1 above)
Comment (chriku) [reply github/src/main/kotlin/gropius/sync/github/Outgoing.kt]: 9a26f69d17142b37ebdf4b1aeee8efbaa0d61920

File: github/src/main/kotlin/gropius/sync/github/Outgoing.kt (hunk 3 above)
Comment (nk-coding) [reply github/src/main/kotlin/gropius/sync/github/Outgoing.kt:352]: then flatmap

File: github/src/main/kotlin/gropius/sync/github/Outgoing.kt (hunk 2 above)
Comment (chriku) [reply github/src/main/kotlin/gropius/sync/github/Outgoing.kt]: 492363db92e638420d8eb650c9efd33d1c25dfc0

File: github/src/main/kotlin/gropius/sync/github/Outgoing.kt (hunk 3 above)
Comment (chriku) [reply github/src/main/kotlin/gropius/sync/github/Outgoing.kt]: d43c14e49da24e0043b1b985422fbf6633db585b

**Review by nk-coding — APPROVED** [review]

**Review by ModellbahnFreak — APPROVED** [review]

**Pull request merged**